src/services/content.service.ts

```
@@ -32,10 +32,10 @@ export class contentService {
     return await this.content.countDocuments().exec();
   }
 
-  async readById(id): Promise<content> {
-    return await this.content.findById(id).exec();
+  async readById(id:any): Promise<content> {
```

ESLint

[error] Insert `·`

(prettier/prettier)